requirements/cuda.txt

```diff
@@ -1,2 +1,2 @@
 # Extra dependencies for NVIDIA CUDA
-instructlab-training[cuda]>=0.8.0
```

Comment:
Are these changes here and below needed? Seems wrong. Could you please explain or revert?